package.json

```diff
@@ -16,6 +17,7 @@
     "@pinia/nuxt": "^0.4.2",
     "@tailwindcss/forms": "^0.5.3",
     "@types/chart.js": "^2.9.37",
+    "@vue/cli-service": "^5.0.8",
```

Comment:
why is @vue/cli-service needed

Reply:
when I try to run it showing error regarding to this . to resolve that I had to install this package